lib/TextDocument/LocationRange.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Phpactor\TextDocument;
+
+class LocationRange
```

Comment:
do we need to introduce this? `Location` can have the `$range` as with the LSP Location

Reply:
Should I extend the Phpactor `Location` class for this? Because it only as a URI and an offset. And I don't think that we want to replace the Phpactor `Location` with the LSP `Location` class right?

Reply:
So a Location should probably have a range and not refer to a single offset, so this is why this may be a can of worms.

Reply:
Sounds easy enough. Then I will update the `Location` class of phpactor to always use a range. This is the cleaner solution anyways if you ask me. That way you are forced to provide the size of the text object you are referencing.